app/lib/features/member/widgets/user_builder.dart

```diff
@@ -207,14 +213,22 @@ class UserBuilder extends ConsumerWidget {
 class UserStateButton extends ConsumerWidget {
   final String userId;
   final Room room;
+  final Task? task;
 
-  const UserStateButton({super.key, required this.room, required this.userId});
+  const UserStateButton({super.key, required this.room, required this.userId, this.task});
 
   Future<void> _handleInvite(BuildContext context) async {
     final lang = L10n.of(context);
     EasyLoading.show(status: lang.invitingLoading(userId), dismissOnTap: false);
     try {
-      await room.inviteUser(userId);
+      if (task != null) {
+        final invitationsManager = await task?.invitations();
```

Comment:
we are showing this as a state already ... sure we don't have a provider already giving us the `InvitationsManager` for the object-id in question?